### ISSUE TITLE
Look into improving html layout on mobile devices

### ISSUE THREAD

**Issue by qu1ck** [openscopeproject/InteractiveHtmlBom#10]
Rendering on narrow screens is a major PITA. You can't discern anything useful on pcb render on tiny screen but at least the top buttons should be visible so that you could switch to BOM only mode.

**Comment by Sembazuru** [comment]
I haven't had too much problem seeing a board on my phone, but I've got a large screen (Pixel-xL), more like a phablet.

Here is a board in Android Chrome that I have with my phone in landscape, no other adjustment.
![screenshot_20180728-143745](https://user-images.githubusercontent.com/4469286/43359699-bf024a76-9274-11e8-8ff6-036fc47a475a.png)


Here is the same board (also in Chrome) where I selected just the top of the board, changed the frame setup for portrait, and then in Chrome dragged the frame separator to trade white space around my board for space for the list.
![screenshot_20180728-143651](https://user-images.githubusercontent.com/4469286/43359701-c27662be-9274-11e8-97a6-2993c1e333ee.png)

Ok, I can agree that the reference designators on the board view can be hard to read, but there are plenty of other landmarks to find where the red square on the screen refers to on the actual board.

My big issue is it is easier to get access to the layout if it is all a single file. I'll open up another issue for that because it doesn't actually fit this one well.

**Comment by qu1ck** [comment]
Yes, as it is viewing bom on a phone is not impossible, it just is not optimized at all.
For example on my moto g6 (1080x2160) screen is not wide enough to even show the layout buttons in portrait mode, you have to go landscape. And fonts are way bigger than necessary on high ppi screens, wasting space.
